rules/windows/process_creation/proc_creation_win_susp_del.yml

```diff
@@ -23,7 +23,7 @@ detection:
             - 'del *.dll'
```

Comment:
can split the 'del *.dll' to
```
- 'del '
- '\*.dll'
```
Need to escape the *  as we are looking for the string *.dll